.github/workflows/haskell.yml

```diff
@@ -39,7 +39,7 @@ jobs:
       run: |
         case "$OS" in
           Windows_NT)   echo "CABAL_VERSION=3.4.0.0-rc5"  >> $GITHUB_ENV;;
-          *)            echo "CABAL_VERSION=3.4.0.0-rc4"  >> $GITHUB_ENV;;
+          *)            echo "CABAL_VERSION=3.4.0.0"      >> $GITHUB_ENV;;
```

Comment:
Do we still need a special version for Windows? Would be nice to use the same version of all platforms.

Reply:
We still do unfortunately.  We use `setup-haskell` uses Chocolatey to install `cabal` on Windows instead of `ghcup` and Chocolatey hasn't updated yet.